policies/en-US/WindowsTerminal.adml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--  (c) 2024 Microsoft Corporation  -->
+<policyDefinitionResources xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" revision="1.0" schemaVersion="1.0" xmlns="http://schemas.microsoft.com/GroupPolicy/2006/07/PolicyDefinitions">
+  <displayName>Windows Terminal</displayName>
+  <description>Windows Terminal</description>
+  <resources>
+    <stringTable>
+      <string id="WindowsTerminal">Windows Terminal</string>
+      <string id="SUPPORTED_WindowsTerminal_1_21">At least Windows Terminal 1.21</string>
+      <string id="DisabledProfileSources">Disabled Profile Sources</string>
+      <string id="DisabledProfileSourcesText">Profiles will not be generated from any sources listed here. Source names can be arbitrary strings. Potential candidates can be found as the "source" property in Windows Terminal's settings.json file.

```

Comment:
What happens with already generated profiles? Are they still usable? Then please add a small note about this fact.

And as I read the original issue the author is asking for disabling the profile. So it should disable/remove already existing profiles. => If they are still be visible in settings and disabled there should be a hint like "This profile is disabled by your administrator."

If disabling existing profiles is out of scoppe or not expected for this policy a second true/false policy "disable existing profiles from blocked sources" might be a solution.